experiments/neural_mcts/eval_scripts/plotting/utils.py

```diff
@@ -0,0 +1,70 @@
+import re
+
+
+def get_model_and_epoch(model_line):
```

Comment:
Are you parsing the command line to get the model and epoch? Or is this parsed from the log string?

Why not just grab these values from the arguments json or something? Parsing is brittle

Reply:
yes I am parsing the CLI output. I bet there could be cleaner solutions. Do you have any suggestions?

Reply:
tbh I haven't looked at this code in a little while and I'm not sure why it's apparently not used.